index.d.ts

```diff
@@ -672,6 +675,21 @@ declare global {
     FramePrototypingMixin,
     ReactionMixin {}
 
+  interface OpaqueNodeMixin  {
+    readonly id: string
+    readonly absoluteTransform: Transform
+    relativeTransform: Transform
+    x: number
+    y: number
+    readonly width: number
+    readonly height: number
+    readonly removed: boolean
+    remove(): void
+    readonly parent: (BaseNode & ChildrenMixin) | null
+    name: string
+    toString(): string
+  }
```

Comment:
The order of these matches the [properties for the trait](https://github.com/figma/figma/blob/master/web/js/figma_app/plugin/jsvm_public_node.ts#L53) to make it easier to scan visually.